src/template.html

```diff
@@ -2,14 +2,42 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <script src="https://kit.fontawesome.com/70657c6fc0.js" crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
-
-    <title>To do list</title>
+    <title>Capstone project</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/fe8431d094.js" crossorigin="anonymous"></script>
 </head>
 <body>
-<h1>Hello World</h1>
+    <header>
+        <nav>
+           <div class="logo">
+            <h2>Mnet</h2>
+           </div> 
+           <ul>
+            <li>
+                <a href="">movies(<span class="total">0</span>)</a>
+            </li>
+            <li>
+                <a href="">new movies</a>
+            </li>
+            <li>
+                <a href="">old movies</a>
+            </li>            
+           </ul>
+        </nav>
+    </header>
+
+    <main>
+        <ul class="wrap">
+           
+        
+        </ul>    
+    </main>
+
+    <footer>
+        <h3>created by Microverse under CC license</h3>
+    </footer>
+    <script type="module" src="index.js"></script>
```

Comment:
Please remove the <script> from template.html. It is not supposed to be there since you are using webpack.